File: java/src/main/java/type/StringType.java
package type;

public class StringType {

    public static void main(String[] args) {
        Person person = new Person("김영재", 25);
        String log = String.format("이름: %s, 나이: %d", person.getName(), person.getAge());

        StringBuilder sb = new StringBuilder();
        sb.append("이름: ");
        sb.append(person.getName());
        sb.append(System.lineSeparator());
        sb.append("나이: ");
        sb.append(person.getAge());
        System.out.println(sb);

        String str = "ABC";
        System.out.println(str.charAt(0));
    }

}

File: kotlin/src/main/kotlin/section1/type/StringType.kt
package section1.type

fun main() {
    val person = Person("김영재", 25)
    val log = "이름: ${person.getName()}, 나이: ${person.getAge()}"

    val output = """
        이름 : ${person.getName()}
        나이 : ${person.getAge()}
    """.trimIndent()
    println(output)

    val str = "ABCDE"
    println(str[0]) // 인덱싱 가능
}
